acceptance/script.prepare

```diff
@@ -93,3 +93,7 @@ envsubst() {
     # when MSYS_NO_PATHCONV is enabled.
     env -u MSYS_NO_PATHCONV envsubst.py
 }
+
+print_telemetry_bool_values() {
```

Comment:
Can we print both the key and the value here? Sometimes the value can be false, which is semantically important.

Reply:
Good idea, updated.